index.html

```diff
@@ -16,7 +16,7 @@
     </p>
     <p>
       There's a reason why Emacs was
-      <a href="http://jtnimoy.net/item.php?handle=14881671-tron-legacy" title="Emacs
+      <a href="http://ergoemacs.org/emacs/emacs_in_tron.html" title="Emacs
```

Comment:
That's a shame the site is down :(
How about we link to the wayback machine version instead?
```suggestion
      <a href="https://web.archive.org/web/20200910190321/http://www.jtnimoy.net/item.php?handle=14881671-tron-legacy" title="Emacs
```